core/config.go

```diff
@@ -53,6 +54,21 @@ func (b BodyConfig) CreatePrompt(stdinReader io.ReadCloser) Prompt {
 	return p
 }
 
+type NewLinesConfig struct {
+	BeforeVersion    int64 `yaml:"beforeVersion,omitempty"`
```

Comment:
I would use `int` instead of int64 here, makes it simpler later and we don't need that much precision for new lines, we are expecting values between like 0 and 2.